src/test/scala/io/iohk/ethereum/vm/CreateOpcodeSpec.scala

```diff
@@ -92,7 +92,7 @@ class CreateOpcodeSpec extends WordSpec with Matchers {
 
         newAccount.balance shouldEqual fxt.endowment
         result.world.getCode(fxt.newAddr) shouldEqual fxt.contractCode.code
-        result.world.getStorage(fxt.newAddr).load(0) shouldEqual 42
+        result.world.getStorage(fxt.newAddr).load(UInt256(0)) shouldEqual UInt256(42).toBigInt
```

Comment:
Looks like an artifact of the previous refactoring: `.load(0) shouldEqual BigInt(42)`